web-main/src/app/state/core/conversation/conversation.actions.ts

```diff
@@ -103,6 +103,19 @@ export namespace Con {
         })
       }
     }
+    //TODO: this needs sendMessage: started, succeeded, failed actions
+
+    //one action
+    export namespace Subscriptions {
+      export const SOURCE = common.Action.Source.from(Api.SOURCE, 'Subscriptions')
+      export const actions = ngrxStore.createActionGroup({
+        source: SOURCE,
+        events: {
+          'addMessage': ngrxStore.props<{ message: models.Conversation.Message.InContext.Input}>(),
```

Comment:
`addMessage` is more command based approch rather then event, based, asko your self a question, what happend? what event we are trying to handle, we are trying to handle "messageRecived" event.